jupyterlab_comments/jupyterlab_comments/handlers.py

```diff
@@ -39,6 +41,15 @@ class DetachedCommentsHandler(APIHandler):
   def get(self):
     file_path = self.get_argument('file_path')
     current_path = self.get_argument('current_path')
+
+    if current_path.startswith("~"):
```

Comment:
is it always guaranteed that file_path and current_path return a value? otherwise you may want to do raise exceptions.